Sources/Apollo/InterceptorRequestChain.swift

```diff
@@ -23,13 +23,9 @@ final public class InterceptorRequestChain: Cancellable, RequestChain {
   private var interceptors: [ApolloInterceptorReentrantWrapper]
   private var callbackQueue: DispatchQueue
   @Atomic public var isCancelled: Bool = false
+  @Atomic private var isFinished: Bool = false
```

Comment:
I added this because `cancel` could still be called after an operation had completed which would result in `managedSelf` having unbalanced `release` calls.